lib/mpl_toolkits/axes_grid1/tests/test_axes_grid1.py

```diff
@@ -707,3 +707,36 @@ def test_removal():
     fig.canvas.draw()
     col.remove()
     fig.canvas.draw()
+
+
+@image_comparison(['anchored_locator_base_call.png'], style="mpl20", remove_text=True)
```

Comment:
I also wonder if we need to use a full image comparison. Would a `check_figure_equal` work where one is a plain figure, and the other contains a single subfigure?

Reply:
@QuLogic , i added the default code i had missed in your message a couple days ago. I also tried your idea regarding the check_figure_equal. If i interpreted your suggestion properly, the problem doesn't occur unless you have multiple subfigures. IE `fig1= fig.subfigures(nrows=1, ncols=1)` and then create one without subfigures for comparison.

I did remove all of the colorbar code, though, because it wasn't needed to replicate the bug.

Reply:
Ah, if you need multiple subfigures, then this will be difficult to re-create for `check_figures_equal`, at least simply.